Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,133 @@
+class Supervisor < Formula
+  homepage "http://supervisord.org/"
+  url "https://pypi.python.org/packages/source/s/supervisor/supervisor-3.1.3.tar.gz"
+  sha256 "e32c546fe8d2a6e079ec4819c49fd24534d4075a58af39118d04367918b3c282"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "setuptools" do
+    url "https://pypi.python.org/packages/source/s/setuptools/setuptools-15.0.tar.gz"
+    sha256 "718d13adf87f99a45835bb20e0a1c4c036de644cd32b3f112639403aa04ebeb5"
+  end
+
+  resource "meld3" do
+    url "https://pypi.python.org/packages/source/m/meld3/meld3-1.0.2.tar.gz"
+    sha256 "f7b754a0fde7a4429b2ebe49409db240b5699385a572501bb0d5627d299f9558"
+  end
+
+  resource "elementree" do
+    url "http://effbot.org/media/downloads/elementtree-1.2.6-20050316.tar.gz"
+    sha256 "b29d5f2417cb331562c8c5f8ebef2a895ba540261c4245f526143c6a31bccb04"
+  end
+
+  def install
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+
+    resources.each do |r|
+      r.stage { system "python", *Language::Python.setup_install_args(libexec/"vendor") }
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    # Can't use `setup_install_args` - http://github.com/Supervisor/supervisor/issues/608
+    system "python", "setup.py", "install", "--prefix=#{libexec}", "--record=installed.txt"
+
+    bin.install Dir[libexec/"bin/supervisor*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+    (prefix/"supervisord.conf").write supervisord_conf
```

Comment:
This is so the distributed config is written out somewhere and the running config doesn't get blatted during an upgrade. Assuming upgrades go through the same install process?